frontend/src/App.svelte

```diff
@@ -101,6 +116,7 @@
       <Date />
 
       <div class="settings">
+        Search: <input type="search" placeholder="..." bind:value={searchText} />
```

Comment:
I would move `Search:` into placeholder so it doesnt take a lot of space - how it looks on phones anyway?

We can also add `autofocus` so its automatically focused on page load or activated by `ctrl+f` or `/`.